superset/connectors/base/models.py

```diff
@@ -248,10 +249,10 @@ def data(self) -> Dict[str, Any]:
         for column_name in self.column_names:
             column_name = str(column_name or "")
             order_by_choices.append(
-                (json.dumps([column_name, True]), column_name + " [asc]")
+                (json.dumps([column_name, True]), column_name + __(" [asc]"))
```

Comment:
Not sure if it makes more sense to leave the space in or out of the translation here... perhaps out? Same issue a couple lines down.